testing/kyotocabinet/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Andrey L <innerspacepilot@gmx.com>
+# Maintainer:  Andrey L <innerspacepilot@gmx.com>
+pkgname=kyotocabinet
+pkgver=1.2.77
+pkgrel=0
+pkgdesc="Kyoto Cabinet is a library of routines for managing a database."
+url="https://fallabs.com/kyotocabinet/"
+arch="all"
+license="GPL-3.0-or-later"
+depends=""
+makedepends="zlib-dev lzo-dev xz-dev"
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="https://fallabs.com/kyotocabinet/pkg/kyotocabinet-$pkgver.tar.gz"
+builddir="$srcdir/kyotocabinet-$pkgver"
+
+build() {
+	cd "$builddir"
+	./configure --prefix=/usr --enable-lzo --enable-lzma
+	make
+}
+
+check() {
+	cd "$builddir"
```

Comment:
needs to do something useful to make sure build gone fine

most of time `make check`